README.md

```diff
@@ -46,26 +46,32 @@ Build the C++ tools:
     make build
 
 ### Step 5: ###
+Alias python to python3 to avoid using python2 accidentally:
+
+    alias python=python3
```

Comment:
Why was this required?  Are there commands in prjxray that still reference python (rather than python3)?

Reply:
You're right, I don't think this is necessary; I will remove it.